### PR TITLE
Document changes for Rocky and add 3.6 roadmap

### PR DESCRIPTION
**Pull request by amanda11**
Part of StackStorm/community#88

### CODE REVIEW

File: docs/source/_includes/runner_parameters/quick_reference.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. table::
+.. table::
```
Comment (cognifloyd) [inline docs/source/_includes/runner_parameters/quick_reference.rst:1]:
```suggestion
.. table::
```

File: docs/source/_includes/available_permission_types.rst [hunk 2]
```diff
@@ -56,8 +56,6 @@ Global
 +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
 | **execution_views_filters_list** | Ability view all the distinct execution filters.                                                                                                          |
 +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
-| **key_value_pair_list**          | Ability to list (view all) Key-Value Pairs.                                                                                                               |
```
Comment (cognifloyd) [inline docs/source/_includes/available_permission_types.rst:59]: These rbac changes are unrelated.
Are they changes in this file Auto generated?

**Review by cognifloyd — COMMENTED** [review]
The Rocky stuff looks good. A couple of questions about interrelated changes.

File: docs/source/_includes/available_permission_types.rst (hunk 2 above)
Comment (amanda11) [reply docs/source/_includes/available_permission_types.rst]: @cognifloyd  Good spot - it was picking up 3.5 st2 source rather than the 3.7dev - re-generated after cleaning up my cloned repo.

**Review by cognifloyd — APPROVED** [review]
